### ISSUE TITLE
Stuck during changing from one chat to another

### ISSUE THREAD

**Issue by Haree16** [eneshecan/whatsapp-for-linux#197]
While changing from one chat to another it is stuck can't do anything need to restart the app once again

**Comment by ocpalo** [comment]
Please provide more information. You can use the following template.

```
**Description**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See the error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment**
 - Distro [e.g. Ubuntu, Arch Linux]
 - Package [e.g. snap, debian]
 - Desktop Environment or Window Manager [e.g. Gnome, i3wm]
```

**Comment by eneshecan** [comment]
Probably related to #181

**Comment by grealish** [comment]
I managed to reproduce this, it's not exactly a duplicate as it doesn't matter if they are GIF's or images as referred to #181
but taking on the advice of that issue, this mitigates the problem 
`WEBKIT_DISABLE_COMPOSITING_MODE=1`

steps to have the issue:
- Start Whatsapp for linux
- wait to load, 
- go to a different chat, group or what ever, click on the chat box and start typing, 
- nothing echos on the text box and it freeze for a min or two, then you can see your text os it maybe not there
- restart.. and it's all the same again :( 

```
$ WEBKIT_DISABLE_COMPOSITING_MODE=1 whatsapp-for-linux 
<13>May  5 10:01:09 whatsapp-for-linux[825556]: 
<13>May  5 10:01:09 whatsapp-for-linux[825556]: ** (whatsapp-for-linux:825555): WARNING **: 10:01:09.307: Error loading plugin: libnuspell.so.5: cannot open shared object file: No such file or directory
<13>May  5 10:01:09 whatsapp-for-linux[825556]: 
<13>May  5 10:01:09 whatsapp-for-linux[825556]: SettingMap: Key file does not have key “close_to_tray” in group “General”, returning default value.
<13>May  5 10:01:09 whatsapp-for-linux[825556]: SettingMap: Key file does not have key “start_in_tray” in group “General”, returning default value.
<13>May  5 10:01:09 whatsapp-for-linux[825556]: SettingMap: Key file does not have key “header_bar” in group “General”, returning default value.
<13>May  5 10:01:09 whatsapp-for-linux[825556]: SettingMap: Key file does not have key “start_in_tray” in group “General”, returning default value.
```

**Comment by eneshecan** [comment]
Fixed by f7eb282
